CONTRIBUTING.md

```diff
@@ -1,74 +1,52 @@
+*Owners* have admin access and are responsible for the management, maintenance, and operations of repositories.
 
-Nominations for Collaborator status happen through the typical Steering Committee decision making process. That is, to nominate one or more Collaborators, Steering Committee then moves to interview the nominees. The nominee is approved or rejected following the same consensus seeking process for new team members.
+### Steering committee
+*Steering committee* members are key *collaborators* who have demonstrated design or technical expertise critical to the driving the FAST-DNA project and community forward.
 
-#### Code Owners
-Collaborators can become a Code Owner for a specific package(s). They have the ability to approve a pull request that has made changes in those package(s) and at least one Code Owner will be required for any package.
-
-Collaborators can be nominated as a Code Owner for a specific package by Steering Committee members and will go through the same process that a Contributor does to become a Collaborator.
+### Collaborators
+*Collaborators* have write access and have an active and sustained impact on the project and participate in triaging issues, reviewing code, mentoring, and working to improve the architectural quality.
 
-The current list of Code Owners is located in [CODEOWNERS](.github/CODEOWNERS).
+### Code owners
+As subject matter experts, *code owners* approve pull requests on the packages they own. There is a required minimum of one code owner for each package. *Code owners* are listed in [CODEOWNERS](.github/CODEOWNERS).
 
 ### Contributors
-Membership for Contributors, having 'read' access, can be any individual who have shown interest by contributing a pull request to the project.
+*Contributors* have read access and can be anyone who has contributed a completed pull request to the project.
 
-Contributors can self-petition the Steering Committee for Collaborator status by written request to be put on the Steering Committee meeting agenda. In order to be considered however, such self-nominations must be sponsored by an existing Steering Committee member, after which it follows the same process as above. To sponsor a nomination, the Steering Committee member must indicate consent.
+### Nominations & apointments
+* To become a *contributor*, a community member must have a pull request approved and mergred into the FAST-DNA project master branch. 
+* To become a *collaborator*, a *contributor* will petition the *steering committee* who will approve or deny the request.
+* To become a *code owner*, a *collaborator* will be (a) nominated by a *steering committe* member or (b) petition the *steering committee* who will approve or deny the request.
+* To join the *steering committee*, a *collaborator* will be nominated by a *steering committe* member and the *steering committee* who will approve or deny the request.
```

Comment:
second use of "committee" spelling issue

Reply:
Fixed